src/main/drivers/vtx_common.c

```diff
@@ -228,4 +228,74 @@ bool vtxCommonLookupPowerValue(const vtxDevice_t *vtxDevice, int index, uint16_t
         return false;
     }
 }
+
+bool vtxCommonSerializeCustomDeviceStatus(const vtxDevice_t *vtxDevice, sbuf_t *dst)
```

Comment:
This should really be declared static (i.e. not available outside of this compilation unit). Big part of the value of `static` is that it means other developers know they don't have to look for calls into this function anywhere outside of the current file.

Turning this on its head and moving the serialisation in here is a good idea. But this means that you are now not actually using the return value, since you are setting the 0 size directly in this function, so the return value can be removed here and in the VTable function.

Reply:
good to know!